modules/ocf_backups/files/upload-to-box

```diff
@@ -164,6 +105,26 @@ def get_access_token(creds):
         return response['access_token']
 
 
+def upload_to_box(creds):
+    """Uploads the archive to box at the configured location"""
+    lftp_cmd = '''
+        set ftps:initial-prot ""
+        set ftp:ssl-force true
+        set ftp:ssl-protect-data true
+        open {box_ftp}
+        user ocf@berkeley.edu {passwd}
+        mirror -R --parallel={concurr} {archive} {folder}
+    '''.format(
+        box_ftp=BOX_FTP,
+        passwd=creds['password'],
+        concurr=MAX_CONCURR_UPLOADS,
+        archive=ARCHIVE,
+        folder=FOLDER,
+    )
+
+    subprocess.run(['lftp', '--norc'], input=lftp_cmd.encode()).check_output()
```

Comment:
I think this can just be `subprocess.check_call(['lftp', '--norc'], input=lftp_cmd.encode())` since you're discarding the output anyway, so collecting it isn't really useful.

Reply:
We actually can't use `check_call`, here, since it  doesn't support the `input` keyword.